### PR TITLE
Update `windows-sys` 0.52

### PR DESCRIPTION
**Pull request by kennykerr**
This is the first update to the `windows-sys` crate in 8 months.  https://github.com/microsoft/windows-rs/releases/tag/0.52.0

### CODE REVIEW

**Comment by Darksonn** [comment]
Tokio will update when mio does.

**Comment by kennykerr** [comment]
Looks like `mio` still has an older MSRV than `tokio` and `windows-sys`. https://github.com/tokio-rs/mio/pull/1725

**Comment by Darksonn** [comment]
Since the mio PR was closed, I'll close this for now.

At the latest, we can update in March when the old LTS is no longer supported.

**Comment by westy92** [comment]
I opened a PR to unblock this.
https://github.com/tokio-rs/mio/pull/1783

**Comment by Thomasdezeeuw** [comment]
After #6635 is merged I think this can be reopened.

**Comment by Darksonn** [comment]
I reopened and rebased this. Let's see if CI passes.

**Review by Darksonn — APPROVED** [review]
Thanks.

**Pull request merged**
